CHANGELOG.md

```diff
@@ -94,7 +94,7 @@
 ### Changed
 
 - ❗️ manifest: Disallow `[badges]` to inherit from `[workspace.package.badges]`.
-  This was considered a bug and should inherit from `[workspace.badges]`.
+  This was considered a bug.
   Keep in mind that `[badges]` is effectively deprecated.
```

Comment:
What about this?

```suggestion
  The support of it was considered a bug, since `[badges]` is deprecated.
```

Either is fine. `r=me` when ready.

Reply:
That implies a causation which isn't there.